react-components/src/query/useSearchMappedEquipmentAssetMappings.tsx

```diff
@@ -119,6 +119,58 @@ export const useAllMappedEquipmentAssetMappings = (
   });
 };
 
+export const useMappingsForAssetIds = (
+  models: AddModelOptions[],
+  assetIds: number[]
+): UseInfiniteQueryResult<InfiniteData<ModelMappingsWithAssets[]>, Error> => {
+  const sdk = useSDK();
+
+  return useInfiniteQuery({
+    queryKey: [
+      'reveal',
+      'react-components',
+      'mappings-for-asset-ids',
+      ...models.map((model) => [model.modelId, model.revisionId]),
+      ...assetIds
+    ],
+    queryFn: async ({ pageParam = models.map((model) => ({ cursor: 'start', model })) }) => {
```

Comment:
Do you need this `models.map(model => ...` here as well as at the bottom of the `useQuery`, in the `initialPageParam` parameter?

Reply:
Right, nice catch